reference/api/keys.md

```diff
@@ -62,6 +62,10 @@ Since the `key` field depends on the master key, it is computed at runtime and t
 **Default value**: N/A
 **Description**: An array of API actions permitted for the key, represented as strings. API actions are only possible on authorized [`indexes`](#indexes). `["*"]` for all actions.
 
+::: warning
+We do not recommend creating keys that can perform all actions as it would almost be the same as the master key.
```

Comment:
I think the problem is not that it becomes similar to the master key (which I actually disagree with as a statement because in the docs we emphasise the master key's main function should be to secure an instance, not to access endpoints), but that the broader a key's permissions, the bigger the security danger they pose. That is, if a key with access to `POST /search` leaks, the damage is fairly small. If a key with `*.*` leaks, someone can easily mess up your instance beyond repair.